src/app/public/modules/popover/popover-content.component.ts

```diff
@@ -216,14 +217,19 @@ export class SkyPopoverContentComponent implements OnInit, OnDestroy {
         this.setupAffixer();
       }
 
-      this.affixer.affixTo(this.caller.nativeElement, {
+      const affixOptions: SkyAffixConfig = {
         autoFitContext: SkyAffixAutoFitContext.Viewport,
         enableAutoFit: true,
         horizontalAlignment: parseAffixHorizontalAlignment(this.horizontalAlignment),
         isSticky: true,
-        placement: parseAffixPlacement(this.placement),
-        verticalAlignment: 'middle'
-      });
+        placement: parseAffixPlacement(this.placement)
+      };
+
+      if (affixOptions.placement === 'left' || affixOptions.placement === 'right') {
+        affixOptions.verticalAlignment = 'middle';
+      }
```

Comment:
Should we also set an `else` default?

Reply:
The else case was just using the defaults. I went ahead and wrote in else cases even though currently they are just the defaults. That will future proof us in case we ever change the defaults for some reason.